scripts/bump_cloudbeat.sh

```diff
@@ -39,65 +47,78 @@ update_version_mergify() {
         title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
 EOF
     git add .mergify.yml
-    git commit -m "Update .mergify.yml"
-
-    gh label create "backport-v$CURRENT_CLOUDBEAT_VERSION"
+    if git diff --cached --quiet; then
+        echo "No changes to commit in .mergify.yml"
+    else
+        git commit -m "Update .mergify.yml"
+        gh label create "backport-v$CURRENT_CLOUDBEAT_VERSION" --force
+    fi
```

Comment:
Can we use `--only` to simplify the statement?
```suggestion
    git commit -m "Update .mergify.yml" --only .mergify.yml && gh label create "backport-v$CURRENT_CLOUDBEAT_VERSION" --force

```

Reply:
no, we can't run `git commit` when there are no changes, as this will return an error code which will exit due to `set -e`, that's why it's wrapped in an if/else